powertools-parameters/src/main/java/software/amazon/lambda/powertools/parameters/AppConfigProvider.java

```diff
@@ -29,6 +29,8 @@
 import software.amazon.lambda.powertools.parameters.cache.CacheManager;
 import software.amazon.lambda.powertools.parameters.transform.TransformationManager;
 
+import static software.amazon.awssdk.utils.StringUtils.isBlank;
```

Comment:
Seems it's not used... can we remove.
Plus I'm not sure `blank` is correct, `empty` might be better.

Reply:
Oops, left that in from an earlier experiment. Have done a bit of refactoring.